roles/provision-unifiedpush-apb/tasks/provision-ups.yml

```diff
@@ -42,6 +42,7 @@
       - >-
         --openshift-sar={"namespace":"{{ namespace }}","resource":"deploymentconfigs","name":"ups","verb":"update"}
       - --http-address=0.0.0.0:{{ ups_proxy_port }}
+      - --skip-auth-regex=/rest/sender,/rest/registry/device,/rest/prometheus/metrics
```

Comment:
Do we really want this for the `metrics` ?

Reply:
Yes, I think its something we can add later where metrics requires basic auth or token/cookie header to be sent too. That's a different problem to solve on the prometheus kubernetes integration side (service annotations)
No auth is easiest to get it working